src/Traits/ModelCaching.php

```diff
@@ -7,12 +7,13 @@
 {
     public static function all($columns = ['*'])
     {
-        if (config('laravel-model-caching.disabled')) {
+        $class = get_called_class();
+        $instance = new $class;
+
+        if (config('laravel-model-caching.disabled') || !$instance->isCachable) {
```

Comment:
The problem is that `isCachable` can never be set for the `all()` method, since setting `isCachable` is set through a query scope, which returns a Builder object, which doesn't have an all method. Consider that the following code throws an exception:
```php
$model->disableCache()->all();
```

Unfortunately the only way to disable caching on `all()` queries is to turn it off in config at this point, or change to `get()`. Perhaps you have some thoughts on how we can disable the cache on the model? In the past, I've had a method on the model, instead of a scope, and that had more problems than it solved, so I moved to scope.